styleguide/_themes/derek/scss/patterns/grid/ias.scss

```diff
@@ -0,0 +1,112 @@
+.ias-container {
+  @include container-fixed;
+  max-width: 960px;
+  padding-bottom: 60px;
```

Comment:
Can you add a `$standard-padding` variable in `_themes/global/scss/modifiers.scss` with a value of `40px 15px 60px;` and then reference it here instead of using top/bottom padding, so there's no ambiguity as to what the padding is here?